_source/_posts/2023-06-08-identity-developer-podcast.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog_post
+title: "Unlock Identity Expertise with Our New Developer Podcast!"
+author: edunham
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Listen and learn about the nuances of Okta's identity ecosystem"
+tags: [announcement, podcast, identity, WebAuthN, passkeys, OIN, enterprise]
+tweets:
+- ""
+- ""
+- ""
+image: blog/identity-developer-podcast/podcast.jpg 
+type: awareness
+---
+
+Do you enjoy podcasts? I find that, like audiobooks, podcasts can transform boring tasks into an opportunity for entertainment and education. 
+
+I'm delighted to announce a new offering from Okta's Workforce Developer Advocacy Team: our monthly podcast! 
+
+## Listen and learn with the Workforce Identity Developer Podcast
+
+The audio version of the Workforce Identity Developer Podcast is available on [Casted](https://listen.casted.us/public/49/Workforce-Identity-Developer-Podcast-4ce90a5f), where you can also explore Auth0's ongoing [Authorization in software podcast](https://authorizationinsoftware.auth0.com/public/49/Authorization-in-Software-f9b69587) and [Identity Unlocked](https://identityunlocked.auth0.com/public/49/Identity%2C-Unlocked.--bed7fada) podcast as well.
+
+
+## WebAuthN & Passkeys
+
+In March, with product manager Megha Rastogi, we discussed what Workforce Identity means and reviewed some of the history of security that got us here. We dig into phishing-resistant MFA and compare the relative benefits and drawbacks of WebAuthN and Passkeys. 
+
+{% youtube <PiY5HDp0ABI> %} 
+
+## SSWS Auth Tokens vs OAuth
+
+In April, with Director of Technical Strategy Matt Egan, we explored the [Okta Integration Network](https://www.okta.com/integrations/) and compared OAuth integration between applications to its predecessor, SSWS tokens. Spoiler alert: Handing out a token that confers a lot of perms indefinitely to anyone who holds it isn't a great idea these days! 
+
+{% youtube <oHvukPsaAko> %}
+
+## Building SaaS for Enterprise Customers
+
+In May, with Product Acceleration Principal Architect Dan Marma, we discussed his recent Devday 2023 talk about [The Enterprise-Ready MVP](https://youtu.be/zL2U7qpdRZY), expanding our definition of "Okta Developers" to include everyone who builds code for customers who use Okta as their identity provider. Dan also shared some stories from the field, and I learned about some extreme use cases that enterprises have demanded from their identity providers and the apps that they use those identity providers to access! 
+
+{% youtube <enWM5ZHnEp4> %}
+
+## What Next?
```

Comment:
```suggestion
## What's next for identity education in the Okta developer podcasts?
```

grammar nit and trying to work in a couple more keywords (taking all the chances we can!). But having trouble figuring out a way to work in "identity" in there too so came up with this. definitely a suggestion though, so feel free to remove